package.json

```
@@ -9,6 +9,7 @@
     "serve": "vue-cli-service serve",
     "test:unit": "vue-cli-service test:unit --coverage",
     "test:e2e": "cypress run",
+    "test:all": "vue-cli-service test:unit --coverage && cypress run",
```

wonderful, this will suffice and give us the freedom to run the tests with different flags perhaps in the future. We might not want got show coverage for example when the tests run back to back because that coverage report just gets swallowed by the output.

you could have also done...
```suggestion
    "test:all": "yarn test:unit && yarn test:e2e",
```

I'll approve and leave it to you to decide but it's fine as is, I just want you to know your options.

---

Thank you, Jamie!  Also, I've made note of the options.